cypress/support/fragments/marcAuthority/marcAuthorities.js

```diff
@@ -860,6 +860,25 @@ export default {
     });
   },
 
+  verifyHeadingTypeAccordionAccordionAndClick: () => {
+    cy.expect(headingTypeAccordion.exists());
+    cy.do(headingTypeAccordion.clickHeader());
+  },
+
+  chooseHeadingType: (headingType) => {
```

Comment:
It the same chooseTypeOfHeading()

Reply:
chooseTypeOfHeading() you can choose even more than one 
chooseHeadingType() you can choose only one 
buy if you think i need to use chooseTypeOfHeading() , i will change